app/assets/stylesheets/walkthrough.scss

```diff
@@ -1,4 +1,28 @@
+@import "mixins";
+
 .walkthrough {
```

Comment:
Could we make this an more explicit id, not a class pls. If it's not scoped within a page, there is too much risk that someone else uses the "walkthrough" class for another meaning.

Something like `#cli-walkthrough` would be good